qforce/forcefield.py

```diff
@@ -387,6 +391,236 @@ def set_charge(self, non_bonded):
+    #         source leaprc.gaff\n# You can source other force fields here if necessary
+    #         loadmol2 {self.mol_name}_qforce{self.polar_title}.mol2
+    #
+    #         loadamberparams {self.mol_name}_qforce{self.polar_title}.frcmod
+    #
+    #         ## Add any other command, for example "solvateoct MOL TIP3PBOX 10.0"
+    #
+    #         MOL = loadPDB 1lyd.solv.pdb
+    #         saveamberparm MOL {self.mol_name}_qforce{self.polar_title}.prmtop
+    #         saveamberparm MOL {self.mol_name}_qforce{self.polar_title}.inpcrd
+    #         savepdb       MOL {self.mol_name}_qforce{self.polar_title}.pdb   """)
+
+##############################################
+#               MOL2   WRITING               #
+##############################################
+
+    def write_mol2_title(self, mol2):
+        mol2.write(LOGO_HASH)
+        mol2.write(f"# Name: {self.mol_name}\n")
+
+    def write_mol2_molecule(self, mol2, topo, terms):
+        mol2.write(f"@<TRIPOS>MOLECULE\n{self.mol_name}\n")
+        n_bonds = 0
+        for bond in terms['bond']:
+            n_bonds = n_bonds + 1
+        mol2.write(f"{self.n_atoms:8d} {n_bonds:8d} {1:8d} {0:8d} {0:8d}\n") ##n_bonds?
+        mol2.write(f"esp")## type of charge
+        mol2.write("\n\n")
+
+    def write_mol2_atom(self, mol2, topo, coords, non_bonded, atom_ids, unique_at):
+        mol2.write(f"@<TRIPOS>ATOM\n")
+        for i_idx, (lj_type, a_name, q, mass) in enumerate(zip(non_bonded.lj_types, self.atom_names,
+                                                           self.q, self.masses), start=1):
+
+            mol2.write(f"{i_idx:8d} {lj_type} {coords[i_idx-1][0]:10.4f} {coords[i_idx-1][1]:10.4f} {coords[i_idx-1][2]:10.4f}")
+            mol2.write(f" {unique_at[i_idx][atom_ids[i_idx]]} {1} {self.mol_name} {q:10.6f}\n")
+
+    def write_mol2_bond(self, mol2, topo, terms, atom_ids, unique_at):
+        n_bonds = 1
+        mol2.write(f"@<TRIPOS>BOND\n")
+        for bond in terms['bond']:
+            ids = bond.atomids + 1
+            mol2.write(f'{n_bonds:>6} {ids[0]:>6}{ids[1]:>6} un \n')
+            n_bonds = n_bonds + 1
+        mol2.write(f"@<TRIPOS>SUBSTRUCTURE\n")
+        mol2.write(f"{1:5} {self.mol_name} {1:8} TEMP {0:>8d} **** **** {0:5} ROOT\n")
+
+##############################################
+#               FRCMOD WRITING               #
+##############################################
+
+    def write_frcmod_mass(self, frcmod, non_bonded, atom_ids, unique_at):
+        frcmod.write(f'{self.mol_name} - frcmod generated by QForce\n')
+        frcmod.write(f'MASS\n')
+        for i, mass in enumerate((self.masses), start=1):
+            frcmod.write(f"{unique_at[i][atom_ids[i]]} {mass}\n")
+
+    def write_frcmod_bonds(self, frcmod, terms, alpha_map, atom_ids, unique_at):
+        frcmod.write("\nBOND\n")
+        for bond in terms['bond']:
+            ids = bond.atomids + 1
+            fconst = bond.fconst  * (0.239005)/2   # kJ/mol/A^2 -> kcal/mol/A^2
```

Comment:
I'd appreciate if you could follow the styling of the rest of Q-Force inline with PEP8.
For inline comments two spaces, #, one space, then your comment. f.e.:
`fconst = bond.fconst  * (0.239005)/2  # kJ/mol/A^2 -> kcal/mol/A^2`
Also maximum line length should be 100 columns.
Applicable for the rest of the file also.

Reply:
Put everything to PEP8 styling, also removed some unnecessary comments